include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -275,6 +259,20 @@ __get_bucket_value(_T __value, ::std::uint32_t __radix_iter)
     return (__value >> __bucket_offset) & __bucket_mask;
 }
 
+template <typename _T, bool __is_comp_asc>
+inline __enable_if_t<__is_comp_asc, _T>
+__get_last_value()
```

Comment:
Why "last value"? Maybe "rest values"?

Reply:
It return only one value, so "rest values" is not a good name either.

Reply:
I think we can leave `__get_last_value` because it returns the last value for type `_T` in a sorted sequences depending on a predicate type.

Reply:
Now I see what you mean. You mean last value for the type and comparator